ckanext/validation/utils.py

```diff
@@ -141,22 +152,6 @@ def run_sync_validation(resource_data):
         resource_data['_success_validation'] = True
 
 
-def delete_validation_report(resource_data):
```

Comment:
Why get rid of this helper?

Reply:
It doesn't make much sense because it duplicated part of the code of the action in which it was used.
But it can be returned if that's preferable.